fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTray.kt

```diff
@@ -48,11 +54,20 @@ fun TabsTray(
         .observeAsComposableState { state -> state.privateTabs }.value ?: emptyList()
     val pagerState = rememberPagerState(initialPage = 0)
     val scope = rememberCoroutineScope()
+    val isInMultiSelectMode = multiselectMode is TabsTrayState.Mode.Select
+
     val animateScrollToPage: ((Page) -> Unit) = { page ->
         scope.launch {
             pagerState.animateScrollToPage(page.ordinal)
         }
     }
+    val onTabClick: ((TabSessionState) -> Unit) = { tab ->
+        if (isInMultiSelectMode) {
+            interactor.onMultiSelectClicked(tab, TABS_TRAY_FEATURE_NAME)
+        } else {
+            interactor.onTabSelected(tab, TABS_TRAY_FEATURE_NAME)
+        }
+    }
```

Comment:
Looking at this now, I feel this could be a good candidate for an optional refactor, to consolidate `onMultiSelectClicked` and `onTabSelected` into a singular `onTabClicked` controller/interactor function.